exercises/hello-world/hello_world_test.f90

```diff
@@ -0,0 +1,16 @@
+! This test was created from ..\..\exercism\problem-specifications\exercises\hello-world\canonical-data.json
+! version: 1.1.0 
+!
+program hello_world_test_main
+  use TesterMain
+  use hello_world 
+
+  implicit none
+
+  !! About the test comments:
```

Comment:
Need to expand on this a little bit, as it is the first time students will see this.  Something small, perhaps even referring back to the "README" file, perhaps mentioning the "Instructions" tab on the web.

Reply:
I tried to resolve this in commit ca5074f. Please review.